src/services/diagram-creation-service/diagram-creation.service.ts

```diff
@@ -90,14 +92,16 @@ export class DiagramCreationService implements IDiagramCreationService {
   }
 
   private getInitialProcessXML(processModelId: string): string {
+    const definitionId = uuid.v4();
+
     return `<?xml version="1.0" encoding="UTF-8"?>
     <bpmn:definitions
       xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL"
       xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
       xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
-      id="Definition_1"
+      id="${definitionId}"
```

Comment:
Ich glaube, wir brauchen doch ein Präfix wegen dieser QName-Konvention
```suggestion
      id="Definition_${definitionId}"
```

Reply:
Denn die dürfen, wenn ich mich recht erinnere, nicht mit einer Zahl anfangen und das kann bei UUIDs ja doch passieren 😅

Reply:
Stimmt das habe ich nicht bedacht 👍 Gut aufgepasst